docs/docs/sourcing-from-ghost.md

```diff
@@ -62,43 +62,63 @@ Once the source plugin is set up, you can use the `createPages` API in `gatsby-n
 There are several ways to structure queries depending on how you prefer to work, but here's a very minimal example:
 
 ```javascript:title=gatsby-node.js
+const _ = require(`lodash`)
```

Comment:
I've simplified gatsby-starter-ghost to not need lodash anymore:

https://github.com/TryGhost/gatsby-starter-ghost/commit/4ddb3bcaa7e10be912f9bd78fd1718c665d47813

The same change can be applied here